core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -7777,7 +7777,7 @@ private void checkLiteral2(String expression, String expected) {
     final String expectedDefault = "INSERT INTO \"SCOTT\".\"DEPT\""
         + " (\"DEPTNO\", \"DNAME\", \"LOC\")\n"
         + "VALUES (1, 'Fred', 'San Francisco'),\n"
-        + "(2, 'Eric', 'Washington   ')";
```

Comment:
Is this the expected result, char type value in calcite seem to require aligned lengths.

Reply:
The types of these fields in the Scott database is actually VARCHAR(20), and VARCHAR types are not supposed to store the padding spaces, unlike CHAR(N) types. So the result is correct. I don't know exactly why the result was wrong before.